README.txt

```diff
@@ -9,9 +9,15 @@ For this plugin to work, the following must be added to your config.inc.php file
 ; Enable iThenticate to submit manuscripts after submit step 4
 ithenticate = On
 
-; Credentials are set by context : specify journal path
+; Credentials can be set by context : specify journal path
 ; The username to access the API (usually an email address)
-username[MyJournal_path] = "user@email.com"
+;username[MyJournal_path] = "user@email.com"
+; The password to access the API
+;password[MyJournal_path] = "password"
+
+; default credentials
+; The username to access the API (usually an email address)
+username = "user@email.com"
 
 ; The password to access the API
-password[MyJournal_path] = "password"
+password = "password"
```

Comment:
I thought that these changes could be sufficient ?

Reply:
Whoops, I must've been looking at a single commit in the PR. Thanks, that's fine!